python/src/deltachat/account.py

```diff
@@ -92,6 +95,17 @@ def __init__(self, db_path, os_name=None, logging=True) -> None:
         hook = hookspec.Global._get_plugin_manager().hook
         hook.dc_account_init(account=self)
 
+    def open(self, passphrase: Optional[str] = None) -> bool:
```

Comment:
Since you added `dc_context_open()`, add `dc_context_is_open()` wrapper as well?

Reply:
ugh??? `is_open()` was there already, I don't understand the question :thinking: